sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -82,12 +82,35 @@ def _validate_constructor_args(model_uri, model_class):
         message.format(model_uri=model_uri, model_class=model_class))
 
 
+def no_gpu_available_warning():
+  logging.warning(
```

Comment:
Let's use logging related to this file.

For example, https://github.com/apache/beam/blob/04fdf0818c2147576ebbd51bc0d5b6d575ed905f/sdks/python/apache_beam/utils/retry.py#L59